From: Robert Luberda <robert@debian.org>
Date: Sat, 25 Oct 2014 16:03:00 +0200
Subject: 41 Fix Functions.html garbage

Functions.html generated by ASDocGen contained garbage.
Commenting out the recursion stack check causes the file
to be generated several times - each generated copy is
the same as previous copy and overwrites it, and the final
file no longer contains null bytes (closes: #757328).
---
 src/ASDocGen/ASDocGen.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ASDocGen/ASDocGen.c b/src/ASDocGen/ASDocGen.c
index bed04b8..e8942d9 100644
--- a/src/ASDocGen/ASDocGen.c
+++ b/src/ASDocGen/ASDocGen.c
@@ -658,9 +658,11 @@ write_options_keywords(const char *source_dir, const char *syntax_dir, SyntaxDef
 		show_error ("Excessive recursion into nested Syntaxes. Syntax doc path = \"%s\".", syntax->doc_path );
 		return;
 	}
+/*
 	for (i = syntax_recursion_level; i >= 0 ; --i)
 		if (syntax_recursion_stack[i] == syntax) 
 			return;
+*/
 	++syntax_recursion_level;
 	syntax_recursion_stack[syntax_recursion_level] = syntax;
 
